student.rb

```diff
@@ -0,0 +1,12 @@
+require_relative './person'
+
+class Student < Person
+  def initialize(classroom)
+    @classroom = classroom
+    super(id, name, age, parent_permission: true)
```

Comment:
- [ ] Kindly consider to change your code taking the following lines in consideration:
  - you need to write `age, name = 'Unknown, parent_permission = true'` in the initialize paarmeters.
  - The super keyword should be on line 5 instead of line 6.
  - You dont need to write id in the super parameters as person's id would be given to student.